lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -293,29 +293,34 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
              }
            }
          ],
+         screen_elevator_id,
          closures
        ) do
-    # If any of a closed elevator's alternates are also closed, the normal summary may not be
-    # applicable.
-    if Enum.any?(closures, fn %Closure{id: id} -> id in alternate_ids end) do
-      @active_summary_fallback
-    else
-      case redundancy do
-        :nearby -> {:inside, summary}
-        :in_station -> {:inside, summary}
+    cond do
+      Enum.any?(closures, fn %Closure{id: id} -> id in alternate_ids end) ->
+        # If any of a closed elevator's alternates are also closed, the normal summary may not be
+        # applicable.
+        @active_summary_fallback
+
+      redundancy in ~w[nearby in_station]a and screen_elevator_id in alternate_ids ->
+        {:inside, "This is the backup elevator"}
+
+      redundancy in ~w[nearby in_station]a ->
```

Comment:
Self Note: `~w[nearby in_station]a` - I definitely need to read up more about atoms 👀

Reply:
`~w` is a "sigil", specifically a [word list](https://github.com/elixir-lang/elixir/blob/v1.17.2/lib/elixir/pages/getting-started/sigils.md#word-lists). The `a` modifier makes it a list of atoms instead of strings.